src/SIL.LCModel.Core/SpellChecking/SpellEngine.cs

```diff
@@ -75,9 +97,16 @@ public bool Check(string word)
 		/// </summary>
 		/// <param name="word"></param>
 		/// <returns></returns>
+		#if __MonoCS__
 		private static byte[] MarshallAsUtf8Bytes(string word)
 		{
 			return Encoding.UTF8.GetBytes(Icu.Normalize(word, Icu.UNormalizationMode.UNORM_NFC) + "\0");
+		#else
+		private static string MarshallAsUtf8Bytes(string word)
+		{
+			byte[] bytes = Encoding.UTF8.GetBytes(Icu.Normalize(word, Icu.UNormalizationMode.UNORM_NFC) + "\0");
+			return Encoding.UTF8.GetString(bytes);
+		#endif
```

Comment:
I think this would be clearer if the close brace `}` were inside the `#if` block